pyAFL/teams/models.py

```diff
@@ -108,8 +108,12 @@ def season_stats(self, year: int):
 
         for table in team_tables:
             if table.find("th"):
-                if self.name in table.find("th").text:
-                    df = pd.read_html(str(table))
+                if self.name != "Adelaide":
+                    if self.name in table.find("th").text:
+                        df = pd.read_html(str(table))
+                else:
+                    if "Adelaide" in table.find("th").text and "Port" not in table.find("th").text:
```

Comment:
Thanks for looking into this bug!

One qu: what about the case where `"Adelaide" in table.find("th").text and "Port" in table.find("th").text`?
Is that case ignored? Should there be an `else` block to match the 2nd `if` block?